IRremoteESP8266.cpp

```diff
@@ -749,10 +749,7 @@ static void ICACHE_RAM_ATTR gpio_intr() {
     irparams.rcvstate = STATE_MARK;
     irparams.rawbuf[irparams.rawlen++] = 1;
   } else {
-    if (now < start)
-      irparams.rawbuf[irparams.rawlen++] = (0xFFFFFFFF - start + now) / USECPERTICK + 1;
-    else
-      irparams.rawbuf[irparams.rawlen++] = (now - start) / USECPERTICK + 1;
+    irparams.rawbuf[irparams.rawlen++] = now - start;
```

Comment:
suggestion/thought: Rather than store the delta, we could change rawbuf[] to uint32_t at the cost of twice the memory. We could then do all the calcs (ie. delta/difference) in the non interrupt code, making this even faster.

but this is fine as is.